ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -212,6 +217,7 @@ public void maybeUpdateView() {
         .setOldController(getController())
         .setImageRequest(imageRequest)
         .setControllerListener(mControllerListener)
+        .setControllerListener(mLoadingControllerListener)
```

Comment:
I'm afraid this will not work. Fresco only supports a single controller listener.

What you can do is define a single 'overall' controller listener that actually gets passed to Fresco. This internally would have a list of listeners. ReactImageView could then offer an addControllerListener method, and maybe also clearControllerListeners.